setup.cfg

```diff
@@ -20,10 +20,17 @@ universal = 1
 add_ignore = D401
 
 [pycodestyle]
-exclude = docs/conf.py
+exclude = docs/conf.py, invenio_app_ils/config.py
 
 [isort]
 skip_glob=node_modules
+# black compatible sorting
```

Comment:
with this settings, no more black/isort conflicts, WDYT?

Reply:
What about [this](https://pycqa.github.io/isort/docs/configuration/black_compatibility/) instead?